hastory-server/src/Data/Hastory/Server/Handler/Entries.hs

```diff
@@ -0,0 +1,12 @@
+module Data.Hastory.Server.Handler.Entries where
+
+import Data.Hastory.Server.Handler.Import
+
+createEntryHandler :: AuthCookie -> SyncRequest -> HastoryHandler NoContent
+createEntryHandler cookie syncReq = do
+  user <- (runDB . getBy $ UniqueUsername (unAuthCookie cookie)) >>= ensureWith err401
+  let serverEntry = toServerEntry syncReq (entityKey user)
+      upsertIfNotExist = upsertBy uniqueContentHash serverEntry []
```

Comment:
nit: you want to switch this around with the next line so that it can be read top to bottom.